fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetadataOps.java

```diff
@@ -186,24 +212,22 @@ public void createTable(CreateTableStmt stmt) throws UserException {
         }
     }
 
-    private static List<FieldSchema> parsePartitionKeys(Map<String, String> props) {
-        List<FieldSchema> parsedKeys = new ArrayList<>();
-        String pkStr = props.getOrDefault("partition_keys", "");
-        if (pkStr.isEmpty()) {
-            return parsedKeys;
-        } else {
-            // TODO: parse string to partition keys list
-            return parsedKeys;
-        }
-    }
-
     @Override
     public void dropTable(DropTableStmt stmt) throws DdlException {
         String dbName = stmt.getDbName();
         ExternalDatabase<?> db = catalog.getDbNullable(stmt.getDbName());
         if (db == null) {
             throw new DdlException("Failed to get database: '" + dbName + "' in catalog: " + catalog.getName());
         }
+        ExternalTable table = db.getTableNullable(stmt.getTableName());
```

Comment:
I think we should check whether table exist in hms, not just in Doris.
Same as `databaseExist()`